src/components/definition-list/dd.component.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import propTypes from '@styled-system/prop-types';
+import PropTypes from 'prop-types';
+import { StyledDd } from './definition-list.style';
+
+const Dd = ({
+  mb = 2,
+  children
+}) => {
+  return (
+    <StyledDd
+      mb={ mb }
+    >
+      {children}
+    </StyledDd>
+  );
+};
+
+Dd.propTypes = {
+  ...propTypes.space,
+  /** Content margin bottom. This value will be multiplied by the theme spacing constant (8) */
```

Comment:
You can remove this proptype definition now as styled-system will handle `mb`

Reply:
✅